components/routing.rst

```diff
@@ -346,6 +346,44 @@ automatically in the background if you want to use it. A basic example of the
     are saved in the ``cache_dir``. This means your script must have write
     permissions for that location.
 
+Unicode routing support
+~~~~~~~~~~~~~~~~~~~~~~~
+
+The routing component supports UTF-8 characters in route paths and requirements. Thanks to the `utf8` route option, you can make Symfony match and generate routes with UTF-8 characters:
+
+.. code-block:: yaml
+
+    # routes.yml
+    route1:
+        path:     /category/{name}
+        defaults: { _controller: 'DefaultController::categoryAction' }
+        requirements:
+          name: ".+"
+        options:
+          utf8: true
+
+In this route, the `utf8` option set to true makes Symfony consider the `.` requirement to match any UTF-8 characters instead of just a single byte character, so the following URLs would match: `/category/日本語`, `/category/فارسی`, `/category/한국어`, etc. In case you are wondering, this option also allows to include and match emojis in URLs.
+
+.. code-block:: yaml
+
+    # routes.yml
+    route2:
+        path:     /default/{default}
+        defaults: { _controller: 'DefaultController::defaultAction' }
+        requirements:
+          default: "日本語|فارسی"
+        options:
+          utf8: true
```

Comment:
Do we really need ? I don't see the gain here when it's already well described [here](http://symfony.com/doc/current/routing.html#routing-examples)

I will do it if you insist, though (sounds like a simple copy/paste of the resource I've linked ^ ).